src/components/Calculator.jsx

```diff
@@ -0,0 +1,145 @@
+import { useState } from "preact/hooks";
+
+export default function Calculator() {
+  const [developersCount, setDevelopersCount] = useState(5);
+  const [buildTimeSaved, setBuildTimeSaved] = useState(50);
```

Comment:
Since the calculations are done with clean builds in mind, I think we should go higher. I'd say 70 % is a good default.